pyproject.toml

```diff
@@ -17,7 +17,8 @@ classifiers = [
 dependencies = [
   "neuroconv",
   "nwbinspector",
-  "ndx-events==0.2.0"
+  "ndx-events==0.2.0",
+  "spikeinterface>=101.1"
```

Comment:
I suggest you to pin to a spikeinterface precise version if they are ones doing the conversion.

Reply:
Do you think I should pin neuroconv as well?